joblib/test/test_logger.py

```diff
@@ -30,3 +32,68 @@ def test_print_time(tmpdir, capsys):
     if not re.match(match, err_printed_text):
         raise AssertionError('Excepted %s, got %s' %
                              (match, err_printed_text))
+
+
+def test_logging_levels(caplog):
+    logger = Logger()
+    warn_msg = "This is a message at logging level WARNING. "
+    info_msg = "This is a message at logging level INFO. "
+    debug_msg = "This is a message at logging level DEBUG. "
+
+    logging.basicConfig(level=logging.INFO)
```

Comment:
I am unsure whether this test has border effects, as the logging API relies on global handlers.
Would it be possible to create a local handler that we remove at the end of the test?

That being said, not completely sure this test is needed, maybe we could just remove it as this tests logger functionalities that are already tested by Python?